TikTokApi/tiktok.py

```diff
@@ -847,12 +847,12 @@ def user_liked(self, userID, secUID, count=30, cursor=0, **kwargs) -> dict:
                 res["itemList"]
             except Exception:
                 logging.error("User's likes are most likely private")
-                return []
+                return response
 
             for t in res.get("itemList", []):
                 response.append(t)
 
-            if not res.get("hasMore", False) and not first:
+            if not res.get("hasMore", False) or not first:
```

Comment:
This change causes the package to only return the first page of results, even if hasMore is true. I think the main improvement you saw was from the line 850 change. If you just revert the `or` back to `and` I'll merge :)